packages/help/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["."],
+  "include": [".", "src/Tooltip.stories.tsx"],
```

Comment:
looks like this is leftover from when we were going to put it in the help package